src/hooks/useDomainCardsID.ts

```diff
@@ -0,0 +1,34 @@
+import {useOnyx} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function useDomainCardsID(policyID: string | undefined) {
```

Comment:
There can technically be multiple domainAccountIDs that we would return here, right? Since there could be multiple domains that have this set as the preferred policy. Maybe it would make sense to call it `useDomainCardIDs` or `useDomainAccountIDs`.

Reply:
ok, so I need to refactor this then, for now it takes the first matching entry

Reply:
I've changed it to FundID like in the other locations, if it's a fundID in the cards_ object 

@puneetlath May I work on refactoring it to return an array of the ids in the second PR for selector? Here, without selector it would make things a bit complicated

Reply:
Sure, that makes sense to me.